packages/auth-otp/plugin.ts

```diff
@@ -31,15 +35,30 @@ interface OtpAuthConfig extends Zemble.GlobalConfig {
    * email, name and two factor code.
     */
   readonly emailHtml?: string
-  readonly handleAuthRequest?: (email: IEmail, twoFactorCode: string, context: Zemble.GlobalContext) => Promise<void> | void
-  readonly generateTokenContents: ({ email }: {readonly email: string}) => Promise<Omit<Zemble.OtpToken, 'iat'>> | Omit<Zemble.OtpToken, 'iat'>
+
+  // SMS
+
+  readonly fromSms?: string
```

Comment:
Would be a nice place for a TypeScript literal :) Something like this:
```
type OneToNine = '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
type PhoneNumber = `+${OneToNine}${number}`

fromSms?: PhoneNumber
```

Could make a more advanced one that checks the length as well, but might not be worth the effort :)

Reply:
Actually, you can enter a string here as well, like "Lifeline" for example, and then the message will come from the string instead of the number.